src/vulkan-renderer/world/cube.cpp

```diff
@@ -216,6 +216,10 @@ bool Cube::is_root() const noexcept {
     return &(*m_parent.lock()) == this;
 }
 
+bool Cube::is_leaf() const noexcept {
+    return m_type != Type::OCTANT;
+}
```

Comment:
This method shouldn't be in this class, it can be done inline everytime, also a single `SOLID` cube would not be a leaf?

Reply:
Yes I will remove it and yes, it would be a leaf.

Reply:
Even more precise: A `SOLID` cube is always a leaf, otherwise it can be a leaf if the grid size (=max subcube iteration depth) is specified.

Reply:
Now that I think about it, it makes even more sense because the `is_leaf()` method is rather confusing: It does not account for the grid size!

Reply:
I removed it. Done.